util/linux_install.sh

```diff
@@ -12,12 +12,14 @@ util_dir=$(dirname "$0")
 
 # For those distros that do not package bootloadHID
 install_bootloadhid() {
-    wget https://www.obdev.at/downloads/vusb/bootloadHID.2012-12-08.tar.gz -O - | tar -xz -C /tmp
-    cd /tmp/bootloadHID.2012-12-08/commandline/
-    if make; then
-        sudo cp bootloadHID /usr/local/bin
+    if command -v bootloadHid >/dev/null; then
```

Comment:
Sorry, got my suggestion wrong on the previous attempt....
```suggestion
    if ! command -v bootloadHID >/dev/null; then
```

Reply:
I missed it, too.
thanks